PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1795,11 +1795,22 @@ bool FindReplaceDlg::processFindNext(const TCHAR *txt2find, const FindOption *op
 			return false;
 		}
 	}
-	else if (posFind == -2) // Invalid Regular expression
-	{
+	else if (posFind < -1) 
+	{ // error
 		NativeLangSpeaker *pNativeSpeaker = (NppParameters::getInstance()).getNativeLangSpeaker();
-		generic_string msg = pNativeSpeaker->getLocalizedStrFromID("find-status-invalid-re", TEXT("Find: Invalid regular expression"));
-		setStatusbarMessage(msg, FSNotFound);
+		generic_string  msgGeneral;
+		if (posFind == -2)

```

Comment:
May be out of scope here, however I would really like to see these -1, -2, etc. magic numbers turned into (meaningfully named) symbolic constants.

Reply:
Replacing this by constants would make indeed more sense, but the goal was not to improve infrastructure. Until now, it's hardcoded -1 and -2 everywhere, and since the good/bad check is now `r >= 0` instead of `(r != -1) && (r >= -2)`, it's improved already.